crates/api/src/types/window_relative_to.rs

```diff
@@ -1,25 +1,24 @@
-use std::fmt;
-
-use serde::de;
+use serde::Deserialize;
 use types::{Object, String as NvimString};
 
 use crate::Window;
 
 #[non_exhaustive]
-#[derive(Clone, Debug, Eq, PartialEq, Hash)]
+#[derive(Clone, Debug, Eq, PartialEq, Hash, Deserialize)]
 /// Specifies what a floating window is positioned relative to.
 pub enum WindowRelativeTo {
     /// Positions the window relative to the global Neovim editor grid.
     Editor,
 
-    /// Positions the window relative to another window.
-    Window(Window),
-
     /// Positions the window relative to the current cursor position.
     Cursor,
 
     /// Positions the window relative to the current mouse cursor position..
     Mouse,
+
+    /// Positions the window relative to another window.
+    #[serde(untagged)]
```

Comment:
It seems like this is making the `open_win_basic_config` test fail on 0.8 and 0.9.